crates/core/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "drawbridge-core"
```

Comment:
Let's name it `drawbridge-name`.

Reply:
hmm, it contains `Service` as well, so not sure that's valid, but I could refactor it into providing a public API adding namespacing, then we go back to having a `Service` in top-level `src/lib.rs`?

Reply:
@rvolosatovs I don't understand why that isn't valid.

Reply:
Because then we have a `drawbridge_name::Service`, which handles tags, trees and, soon, auth rather than just handling naming as the crate name would imply

Reply:
But a namespace **contains** tags, trees and so on. And actually, auth will probably wrap namespace. I see:

```mermaid
graph TD
main --> auth;
auth --> name;
name --> tags;
name --> tree;
tree --> hash;
tags --> hash;
```

Reply:
Changed to `name`, although how about `repo`?